src/Microsoft.Data/PrimitiveColumnContainer.cs

```diff
@@ -77,6 +77,13 @@ public PrimitiveColumnContainer(IEnumerable<T> values)
             }
         }
 
+        internal PrimitiveColumnContainer(Memory<byte> memory, Memory<byte> nullBitMap, int length)
+        {
+            Buffers.Add(new DataFrameBuffer<T>(memory, length));
+            NullBitMapBuffers.Add(new DataFrameBuffer<byte>(nullBitMap, length));
+            Length += length;
```

Comment:
This is a constructor, right? `Length` should be `0` to start, so this should be:

`Length = length`.